netbout-web/src/main/java/com/netbout/rest/RqAlias.java

```diff
@@ -111,7 +112,16 @@ public Alias alias() throws IOException {
                 "/login/start"
             );
         }
-        return aliases.iterate().iterator().next();
+        final Alias alias = aliases.iterate().iterator().next();
+        if (alias.photo().equals(Alias.BLANK)) {
```

Comment:
@dmzaytsev also, keep in mind, that in this design we will set user avatar only once. Later, when the user changes the picture in Github, we won't catch that change. We will continue to use the old picture. I think that the only solution is to add randomization here (pseudo code):

```
if (alias.photo().equals(Alias.BLANK) || new Random().nextInt(10) == 5) {
  // ...
}
```